flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -1027,15 +1027,15 @@ under the License.
 	<profiles>
 		<!-- Activate these profiles with -Phive-x.x.x to build and test against different Hive versions -->
 		<profile>
-			<id>hive-2.3.6</id>
+			<id>hive-2.3.9</id>
 			<properties>
-				<hive.version>2.3.6</hive.version>
+				<hive.version>2.3.9</hive.version>
 			</properties>
 			<dependencies>
 				<dependency>
 					<groupId>org.apache.orc</groupId>
 					<artifactId>orc-core</artifactId>
-					<version>${hive-2.3.6-orc-version}</version>
+					<version>${hive-2.3.9-orc-version}</version>
```

Comment:
This complete profile seems to be broken. I neither see `${hive-2.3.6-orc-version}` defined somewhere nor `${hive-2.3.9-orc-version}`. Do I miss something here?

Reply:
Good one, I don't see it either. Looking back through the Git commits, I'm suspecting that https://github.com/apache/flink/pull/13078/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R150 originally introduced this value as  `hive-2.2.0-orc-version`. Probably during a search and replace that one got moved around. 

My guess is that we can completely remove this profile and the `hive-2.2.0-orc-version` from the main POM. WDYT?

Reply:
I suggest to still keep it for test purpose . 
It enables the developer to test againset the corresponding Hive version to make sure it works for the specific version.